charts/outline/templates/001-deployment.yaml

```diff
@@ -16,6 +16,23 @@ spec:
         app: {{ .Release.Name }}
         release: {{ .Release.Name }}
     spec:
+      securityContext:
+        fsGroup: 1001
+        runAsUser: 1001
+        runAsGroup: 1001
```

Comment:
Does the deployment face any issues without these permissions?

Reply:
yes no permissions to write to local storage

Reply:
Can we make this optional? Only when it's local storage. The other stuff looks good.

Reply:
@nushkovg done